### PR TITLE
Issue 1567

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**
Issue fix: #1567

### CODE REVIEW

**Review by SarfrazS — CHANGES_REQUESTED** [review]
Not fixed
Last commit:-
image

image
Now click on view and then click on customer details
image

File: app/Http/Controllers/User/ClientController.php [hunk 1]
```diff
@@ -328,12 +329,12 @@ public function edit($id)
             //for display
             $timezones = array_column($display, 'name', 'id');
             $state = getStateByCode($user->state);
-            $managers = User::where('role', 'admin')
-            ->where('position', 'manager')
-            ->pluck('first_name', 'id')->toArray();
-            $acc_managers = User::where('role', 'admin')
-            ->where('position', 'account_manager')
-            ->pluck('first_name', 'id')->toArray();
+            $managers = User::select(DB::raw('CONCAT(first_name, " ", last_name) As first_name'), 'id')
```
Comment (sandesh556) [inline app/Http/Controllers/User/ClientController.php:332]: Need to think of a better way than using raw queries

**Review by sandesh556 — CHANGES_REQUESTED** [review]

**Review by sandesh556 — APPROVED** [review]

**Review by SarfrazS — APPROVED** [review]
image
image

**Review by bindu95 — APPROVED** [review]
**BEFORE FIX**
_EDIT USER:_
Screenshot 2022-05-16 at 4 15 54 PM

_VIEW USER:_
Screenshot 2022-05-16 at 4 16 14 PM


**AFTER FIX**
_EDIT USER:_
Screenshot 2022-05-16 at 4 17 49 PM

_VIEW USER:_
Screenshot 2022-05-16 at 4 16 14 PM

**Review by RafficMohammed — APPROVED** [review]

**Pull request merged**